docs/how-it-works.mdx

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 2
+---
+
+# How it Works
+
+Learn the high-level workings of Divvi Up and how it accomplishes the
```

Comment:
I ended up going with this

> # How It Works
> 
> Learn how Divvi Up works and how it protects privacy.
> 
> ## Breakdown
> 
> Divvi Up ingests measurements and aggregates them into metrics, while protecting the privacy of each individual measurement. Its functionality can be broadly broken down into several steps.

I prefer the terse, active voiced headline--note that docusaurus implicitly uses this as the page description, but I was otherwise missing a short description of what precisely Divvi Up does.